### DISCUSSION TITLE
Skip missed cronjobs and add delay for cronjobs on server start

### DISCUSSION THREAD

**Discussion by dr460nf1r3** [woodpecker-ci/woodpecker#2199] [Request a Feature]
### Component

server

### Describe the bug

Currently, it seems that missed cronjobs are getting executed when the instance starts up the next time. 

In the environment I'm currently using Woodpecker, it is utilized to run Ansible playbooks at given times. This also includes scheduled server updates & reboots. 

In the case of having the Woodpecker instance down & started up afterward, all those jobs are getting executed at once, causing servers to reboot when they absolutely should not. 

I'm not sure whether this is an actual bug or just how the cron feature works, though this is kind of a dangerous behavior depending on what kind of tasks are dedicated to Woodpecker. 

The proposal is to either change the behavior to not run missed crons at startup or to provide a way to turn it off. 

### System Info

```shell
{"source":"https://github.com/woodpecker-ci/woodpecker","version":"1.0.0"}
```

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't [already an issue](https://github.com/woodpecker-ci/woodpecker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Checked that the bug isn't fixed in the `next` version already [https://woodpecker-ci.org/faq#which-version-of-woodpecker-should-i-use]
- [X] Check that this is a concrete bug. For Q&A join our [Discord Chat Server](https://discord.gg/fcMQqSMXJy) or the [Matrix room](https://matrix.to/#/#woodpecker:matrix.org).

**Comment by 6543** [comment]
hmm so you propose an enhancement, to skip missed crone execs ... and delay cron job init on startup ?

**Comment by dr460nf1r3** [comment]
If you'd label it as enhancement, then yes, I'm proposing this to take away some of the destructiveness of cronjobs running at the wrong times.
